production/cm_tools/cm_tools/geojson_defined_tasks.py

```diff
@@ -21,21 +21,21 @@ def gen_args():
     )
     parse.add_argument(
         "--sqs",
-        default="deafrica-dev-eks-stats-geomedian-semiannual",
-        help="default sqs is the semiannual one",
+        default="deafrica-dev-eks-stats-crop-mask",
```

Comment:
remove default value? this is one of the require field that needed input form your job configuration based on the environment it runs...